into/backends/spark.py

```diff
@@ -0,0 +1,43 @@
+class Dummy(object):
+    pass
+
+try:
+    from pyspark import SparkContext
+    import pyspark
+    from pyspark import RDD
+    from pyspark.rdd import PipelinedRDD
+    from pyspark.sql import SchemaRDD
+    RDD.min
+except (AttributeError, ImportError):
+    SchemaRDD = PipelinedRDD = RDD = SparkContext = Dummy
+    pyspark = Dummy()
+
+
+from collections import Iterator
+
+from datashape import var
+
+from ..into import convert, append
+from ..core import discover
+
+
+@append.register(SparkContext, (list, tuple, Iterator))
+def iterable_to_spark_context(sc, seq, **kwargs):
+    return sc.parallelize(seq)
+
+
+@append.register(RDD, (list, tuple))
+def sequence_to_rdd(rdd, seq, **kwargs):
+    # Hm this seems anti-pattern-y
+    return append(rdd.context, seq)
```

Comment:
Suggest we don't do this.  We're not really adding to the existing rdd.  This should probably just err the same way we would err if we tried to append to a tuple.